src/garage/np/algos/rl_algorithm.py

```diff
@@ -9,10 +9,17 @@ class RLAlgorithm(abc.ABC):
         If the field sampler_cls exists, it will be by LocalRunner.setup to
         initialize a sampler.
 
+    Args:
+        env_spec (garage.envs.EnvSpec): Environment specification.
+        discount (float): Discount.
     """
 
     # pylint: disable=too-few-public-methods
 
+    def __init__(self, env_spec, discount):
```

Comment:
I think it would be best for `RLAlgorithm` to not have any constructor. Just copy the two lines into every class that needs them.

Reply:
I agree